app/ide-desktop/lib/dashboard/src/layouts/Editor.tsx

```diff
@@ -38,6 +39,7 @@ export default function Editor(props: EditorProps) {
   const gtagEvent = gtagHooks.useGtagEvent()
   const gtagEventRef = React.useRef(gtagEvent)
   const remoteBackend = remoteBackendProvider.useRemoteBackend()
+  const { backend } = backendProvider.useStrictBackend()
```

Comment:
not sure this works - it's possible to have a Local project opened while the Cloud Backend is being used for the Drive view.

unfortunately i can't think of an easy fix... might need to add `backend` to `projectStartupInfo` (or perhaps lift the entire `renameProject` function into `projectStartupInfo` - although given that it is potentially set from multiple places it's probably unnecessarily duplicating the logic)